### PR TITLE
chore: Extract line_agg from file source

### PR DESCRIPTION
**Pull request by MOZGIII**
This PR extracts the `line_agg` module from the `file` source, and renames the standalone `file/mod.rs` into `file.rs`.

Pretty trivial change.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -45,8 +46,7 @@
 /src/sinks/util/auto_concurrency @bruceg
 
 /src/sources/docker.rs @LucioFranco
-/src/sources/file/mod.rs @LucioFranco
-/src/sources/file/line_agg.rs @MOZGIII
+/src/sources/file.rs @LucioFranco
```
Comment (bruceg) [inline .github/CODEOWNERS:49]: This can probably be dropped, as he is no longer part of the team.

Comment (MOZGIII) [reply .github/CODEOWNERS:49]: True. I don't want this PR to introduce any changes like that though - to keed it simple and get in quickly.
I'd suggest we create a dedicated PR (or a series of dedicated PRs) to drop/reassign ownership.

**Comment by MOZGIII** [comment]
We can think about that. There are pros and cons to that. Since it's not a blocker - let's create an issue to discuss it.

**Comment by MOZGIII** [comment]
I've overlooked that we also need to share the multiline config among the sources - decided to add it here rather than to another PR.
@bruceg sorry to bother, but could you re-review this?

**Comment by bruceg** [comment]
Is this PR blocking something? If not, the nature of this PR seems to be ideal for discussing where it should be moved.

**Comment by MOZGIII** [comment]
> Is this PR blocking something? If not, the nature of this PR seems to be ideal for discussing where it should be moved.

I agree, let's discuss. I need this for my work to add the line aggregation at the docker source, but I can work off this PR's tree for now.

**Comment by MOZGIII** [comment]
Rolled back the multiline extract change - I'll create a follow-up PR with it to address it separately.

**Comment by MOZGIII** [comment]
I misinterpreted the concern regarding discussing the location of the `line_agg`. I'll create a dedicated issue.

**Pull request merged**